src/math/hough_transform_simd.cpp

```diff
@@ -0,0 +1,277 @@
+#include "hough_transform_simd.h"
+#include "../image_function_helper.h"
+#include "../penguinv/cpu_identification.h"
+
+#ifdef PENGUINV_AVX_SET
+#include <immintrin.h>
+#endif
+
+#ifdef PENGUINV_SSE_SET
+#include <emmintrin.h>
+#endif
+
+#ifdef PENGUINV_NEON_SET
+#include <arm_neon.h>
+#endif
+
+#include <algorithm>
```

Comment:
Put please these inclusions just after `#include "hough_transform_simd.h"` line as per practice it is recommended to put std library inclusions first.